### DISCUSSION TITLE
Looking for a Image for the Banapi m2 zero with H3 processor

### DISCUSSION THREAD

**Discussion by miloit** [johang/sd-card-images#35] [General]
Looking for a Image for the Banapi m2 zero with H3 processor. Where can i find it?

**Comment by johang** [comment]
Try the image for H2+.

**Comment by hutchpe** [comment]
The H2+ boot image doesn't work on the Bananapi M2 Zero unfortunately.

I was able to build a boot image which is working perfectly using the following command:
`docker run --rm -v /tmp/sd-images:/artifacts sd-images build-boot banana_pi_m2_zero allwinner-h3 bananapi_m2_zero_defconfig arm-linux-gnueabihf`

I had to modify the Dockerfile to install the python3-setuptools package as it seemed to be missing.

**Comment by johang** [comment]
Strange. That image should be identical to the image for Banana Pi M2 Zero with H2+.

**Comment by OldHeini** [comment]
How i can download an image file?

**Reply by johang** [reply]
https://sd-card-images.johang.se/boards.html

**Comment by jjaard** [comment]
Just tried on 2 Banana Pi M2/P2 Zero's (as they are named the same by Banana Pi - used neofetch to check on Armbian:
OS: Armbian (22.11.0-trunk) armv7l
Host: Banana Pi BPI-P2-Zero / M2-Zero
Kernel: 5.19.6-sunxi
Uptime: 16 days, 17 hours, 56 mins
Packages: 1419 (dpkg)
Shell: zsh 5.8
Terminal: /dev/pts/1
CPU: Allwinner sun8i Family (4) @ 1.008GHz
Memory: 239MiB / 490MiB)

 the [image](https://sd-card-images.johang.se/boards/banana_pi_m2_zero.html) and it does not getting network at all. For example using [ALL-H3-CC H2+ (Tritium)](https://sd-card-images.johang.se/boards/libretech_all_h3_cc_h2_plus.html) image has internet but does not boot fully (as this is not BPi M2Z image which is exoected behaiviour I suppose, sorry for mistakes in logs as they were copied from photo):

  | Couldn't find partition muc o |   |  
  | Can't set block device |   |  
  | zimage: Bad magic! |   |  
  | SCRIPT FAILED: continuing..• |   |  
  | MMC: no card present |   |  
  | ** Bad device specification mn 0 ** |   |  
  | Couldn't find partition mnc 0:1 |   |  
  | MC: no card present |   |  
  | ** Bad device specification mnc 0 ** |   |  
  | Couldn't find partition mnc 0:1 |   |  
  | MMC: no card present |   |  
  | ** Bad device specification mmc 0 ** |   |  
  | Couldn't find partition muc 0:1 |   |  
  | MM: no card present |   |  
  | No EFI system partition |   |  
  | No EFI system partition |   |  
  | Failed to persist EFI variables |   |  
  | BootOrder not defined |   |  
  | EFI boot manager: Cannot load any image |   |  
  | MMC: no card present |   |  
  | ** Bad device specification mc 0 xx |   |  


<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">MMC: no card present</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">** Bad device specification mmc 0 **</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Couldn't find partition mnc 0</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Can't set block device</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">MMC: no card present</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">** Bad device specification mnc 0 xx</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Couldn't find partition mnc o</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Found legacy kernel configuration</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">MMC: no card present</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Couldn't find partition mmc 0:1</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Device 0: unknown device</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">ethernet01c30000 Waiting for PHY auto negotiation to complete. done</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">BOOTP broadcast 1</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">De arming: no co Re nane: ustng Con80008.g</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Using ethernet@1c30000 device</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">TFTP from server 192.168.8.1: our IP address is 192.168.8.11</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Filename</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">'COABOBOB. ing'.</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Load address: 0x42000000</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Loading: T T TITTTITT</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Retry count exceeded: starting again missing environment variable: preuid</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Retrieving file: pelinux.cig/01-02-81-4a-34-e8-df</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Using ethernet01c30000 device</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">IFT from server 192.168.8.1; our IP address is 192.168.8.11</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Filename pxelinux.cfg/01-02-81-4a-34-eB-df' •</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Load address: 0x43200000</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Loading: TITITTTITT</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Retry count exceeded: starting again</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Retrieving file: pelinux.cfg/COABOBOB</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Using ethernet@1c30000 device</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">TFT from server 192.168.8.1: our IP address is 192.168.8.11</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Filename</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">'pxelinux.cfg COASOBOB'.</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Load address: 0x43200000</p>
<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 12.0px Helvetica">Loading: T</p>
